platform-enterprise_versioned_docs/version-24.2/enterprise/advanced-topics/jvm-memory-tuning.md

```diff
@@ -0,0 +1,66 @@
+---
```

Comment:
Rather than have multiple copies of the same text in various versions, is it possible to make these pages DRY and link back to `platform-enterprise_docs/enterprise/advanced-topics/jvm-memory-tuning.md`?